forecasters/zyzzyva/DESCRIPTION

```diff
@@ -0,0 +1,58 @@
+Package: zyzzyva
+Title: The zyzzyva forecaster
+Version: 20200808-215113
+Authors@R: 
+    c(person(given = "Rob",
+           family = "Tibshirani",
+           role = c("cre", "aut"),
+           email = "tibs@stanford.edu"),
+      person(given = "Maria",
+             family = "Jahja",
+             role = c("aut"),
+	     email = "maria@stat.cmu.edu"),
+      person(given = "Addison",
+             family = "Hu",
+             role = c("aut"),
+	     email = "addison@stat.cmu.edu"),	
+      person(given = "Vishnu",	
+             family = "Shankar",	
+             role = "ctb"),		
+      person(given = "Samyak",		
+             family = "Rajanala",	
+             role = "ctb"),		
+      person(given = "Balasubramanian",	
+           family = "Narasimhan",	
+           role = "ctb"))
+Description: The zyzzyva forecaster.
+License: MIT + file LICENSE
+Depends: R(>= 3.5.0)
+Encoding: UTF-8
+Suggests: testthat
+Imports:
+    forecast,
+    evalforecast,
```

Comment:
I think this is just for `evalforecast::get_target_period()` right? I copied it into `evalcast`. Can you switch that (currently still needs the `killcards` branch as remote).

Reply:
done